### ISSUE TITLE
ivy: ngcc fails to compile inline templates with escaped new lines

### ISSUE THREAD

**Issue by cexbrayat** [angular/angular#28843]
# 🐞 bug report

### Affected Package

ngcc (as noted by @JoostK in a comment below, probably also ngtsc and ngc).

### Is this a regression?

Yes, it appeared in `8.0.0-beta.4`, probably a side effect of #28055

### Description

`ngcc` fails to compile an inline template with an escaped new line.

## 🔬 Minimal Reproduction

The issue is reproducible by trying to compile `ng-bootstrap` with `ngcc`:

```
ng new ivy-lexer --defaults --experimental-ivy
# bump all ng deps to 8.0.0-beta.4
npm install --save-exact @ng-bootstrap/ng-bootstrap@4.0.3
$(npm bin)/ivy-ngcc
```

## 🔥 Exception or Error

`ngcc` throws:
```
Errors parsing template: Unexpected closing tag "ngb-datepicker-month-view"
```

when compiling the `fesm5` or `esm5` bundle.

## 🌍  Your Environment

**Angular Version:**
<pre><code>
Angular CLI: 8.0.0-beta.0
Node: 11.9.0
OS: darwin x64
Angular: 8.0.0-beta.4
... animations, common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... router

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.13.1
@angular-devkit/build-angular     0.13.1
@angular-devkit/build-optimizer   0.13.1
@angular-devkit/build-webpack     0.13.1
@angular-devkit/core              7.3.1
@angular-devkit/schematics        8.0.0-beta.0
@angular/cli                      8.0.0-beta.0
@ngtools/webpack                  7.3.1
@schematics/angular               8.0.0-beta.0
@schematics/update                0.14.0-beta.0
rxjs                              6.3.3
typescript                        3.2.4
webpack                           4.29.0
</code></pre>

**Anything else relevant?**

After digging a little, it appears that any template containing an escaped new line just after a tag definition or an attribute value will fail (for example `<t\\n></t>` or `<t a=b\\n></t>`).

Edit: I opened #28844 with unit tests to reproduce and a possible workaround.

**Comment by JoostK** [comment]
Great find! Given that the parser is a shared part of the compiler, ngc (read: View Engine) and ngtsc are probably also affected.

**Comment by cexbrayat** [comment]
@JoostK Right! I updated the issue to mention it. I pushed a PR in #28844 with unit tests to reproduce and a possible workaround...

**Comment by alxhub** [comment]
(this is not a regression as ngcc is not a released component)

**Comment by cexbrayat** [comment]
Fixed by #28978 (hopefully included in `8.0.0-beta.7`)
